worlds/sc2/client_gui.py

```diff
@@ -463,6 +473,39 @@ def get_location_type_title(self, location_type: LocationType) -> str:
         else:
             title += ""
         return title
+    
+    def is_scoutable(self, remaining_locations, mission_available: bool, layout_locked: bool, campaign_locked: bool) -> bool:
+        if self.ctx.mission_order_scouting == 5:
+            return True
+        elif self.ctx.mission_order_scouting == 4 and not campaign_locked:
+            return True
+        elif self.ctx.mission_order_scouting == 3 and not layout_locked:
+            return True
+        elif self.ctx.mission_order_scouting == 2 and mission_available:
+            return True
+        elif self.ctx.mission_order_scouting == 1 and len([loc for loc in remaining_locations if loc[0] in (LocationType.VICTORY, LocationType.VICTORY_CACHE)]) == 0:
+            # Assuming that when a mission is completed, all victory location are removed 
```

Comment:
Use `MissionOrderScouting.option_all`, etc instead of the numbers

Reply:
Applied as suggested.